Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,30 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for(int i = 0; i < 2; i++)
+            {
+                string name;
+                string surename;
+                int age;
+                double weight;
+                double height;
+
+                Console.Write("Enter the person's first name: ");
```

Comment:
Code right now is a blob. Try to put a blank line between unrelated blocks of code (reading, writing and different fields)